airbyte-integrations/bases/connector-acceptance-test/connector_acceptance_test/config.py

```diff
@@ -83,6 +83,7 @@ class DiscoveryTestConfig(BaseConfig):
     backward_compatibility_tests_config: BackwardCompatibilityTestsConfig = Field(
         description="Configuration for the backward compatibility tests.", default=BackwardCompatibilityTestsConfig()
     )
+    skip_primary_keys_data_type_validation: Optional[bool] = Field(False, description="Skip primary keys data type validation")
```

Comment:
Is there a reason why we made this one `skip...`? To make things more consistent, could we instead make this `validate_primary_key_data_type` and default to True.